### PR TITLE
add open graph image tag

### PR DESCRIPTION
**Pull request by ghost**
this should fix the image tag issue.

We should really think about to use another theme for out blog.

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -15,6 +15,11 @@
   {% if page.author.gplus %}<link rel="author" href="https://plus.google.com/{{ page.author.gplus }}">{% endif %}
   <meta property="og:title" content="{% if page.title %}{{ page.title }} {% else %}{{ site.inc.title }}{% endif %}"/>
   <meta property="og:url" content="{{ site.url }}{{ page.url }}"/>
+  {% if page.image %}
+  <meta content="/images/site-logo.svg" property="og:image">
```
Comment (lenn4rd) [inline _layouts/page.html:19]: Shouldn't the `content` value be `page.image`?

Comment (ghost) [reply _layouts/page.html:19]: hmmm - my idea was that the image in any previews is always the upday logo - in this way no one has to think about it. But I am fine with both

**Comment by ghost** [comment]
I'll merge it - can be changed afterwards

File: _layouts/page.html (hunk 1 above)
Comment (lenn4rd) [reply _layouts/page.html:19]: Seems my comment via email was lost, here it is:

Oh ok. Having a screenshot like in Nicola's post breaks the preview anyway, so I'm fine with just showing the upday logo. In this case I'd remove the conditional though and just put the meta tag.

**Pull request merged**